### PR TITLE
front50 Application V2 API

### PR DESCRIPTION
**Pull request by jeyrschabu**
- Refactoring to use front50 V2 application API

### CODE REVIEW

File: orca-applications/src/main/groovy/com/netflix/spinnaker/orca/applications/tasks/DeleteApplicationTask.groovy [hunk 1]
```diff
@@ -29,30 +29,26 @@ class DeleteApplicationTask extends AbstractFront50Task {
...
 
-    front50Service.credentials.findAll { it.global }.collect {
-      it.name
-    }.each { String globalAccountName ->
-      try {
-        def existingGlobalApplication = front50Service.get(globalAccountName, application.name)
-        if (existingGlobalApplication) {
-          outputs.previousState = existingGlobalApplication
+    try {
+      def existingGlobalApplication = front50Service.get(application.name)
```
Comment (ajordens) [inline orca-applications/src/main/groovy/com/netflix/spinnaker/orca/applications/tasks/DeleteApplicationTask.groovy]: Can probably s/existingGlobalApplication/existingApplication

**Comment by ajordens** [comment]
Needs a rebase

**Comment by jeyrschabu** [comment]
@ajordens PTL

**Comment by ajordens** [comment]
LGTM

**Pull request merged**